package/MDAnalysis/core/groups.py

```diff
@@ -3324,6 +3324,82 @@ def write(self, filename=None, file_format=None,
 
         raise ValueError("No writer found for format: {}".format(filename))
 
+    def sort(self, key='ix', keyfunc=None):
+        """Returns a sorted ``AtomGroup`` using a specified attribute as \
+           the key.
+
+        Parameters
+        ----------
+        key: str, optional
+            The name of the ``AtomGroup`` attribute to sort by (e.g. ``ids``,
+            ``ix``. default= ``ix`` ).
+        keyfunc: callable, optional
+            A function to convert multidimensional arrays to a single
+            dimension. This 1D array will be used as the sort key and
+            is required when sorting with an ``AtomGroup`` attribute
+            key which has multiple dimensions. Note: this argument
+            is ignored when the attribute is one dimensional.
+
+        Returns
+        ----------
+        Group
```

Comment:
Looking at the docs (https://mdanalysis--3188.org.readthedocs.build/en/3188/documentation_pages/core/groups.html?highlight=sort#MDAnalysis.core.groups.AtomGroup.sort), this sets the "Return type" to "Group". We probably want to have this be a :class:`AtomGroup`?

```suggestion
        :class:`AtomGroup`
```